printer_mods/LoganFraser/PiPlate/README.md

```diff
@@ -0,0 +1,7 @@
+If you want to mount your Raspberry Pi so the usb and ethernet ports are externally accessable, look no further.  
+
+Supported: Pi 3B/4B and Trident or Voron 2.4 machines.  
+*New* Support added for Voron 2.4 250 size, and all 3 sizes of V2.4r2 
+

```

Comment:
BOM: 
No additional hardware Required. Use stock bolts to attach the skirt and stock m2 screws to attach the pi to plate.